eng/pipelines/templates/stages/archetype-js-release.yml

```
@@ -152,7 +152,8 @@ stages:
                     steps:
                       - checkout: self
                       - pwsh: |
-                          $adjustedName = "${{artifact.name}}".Replace("azure-", "")
+                          [regex]$pattern = "azure-"
```

```suggestion
                          [regex]$pattern = "^azure-"
```

Adding `^` in the front will say only match the front of the string.

---

Actually I think this can be changed to. 
```
$adjustedName = "${{artifact.name}}" -replace "^azure-", ""
```

---

Thanks. Updated the code.